docs/notifications/overview/index.md

```
@@ -258,11 +272,42 @@ For detailed information about template syntax, available data structures, and e
 
 Watchtower uses Shoutrrr's [smtp service](https://shoutrrr.nickfedor.com/services/email/){target="_blank" rel="noopener noreferrer"} to send email notifications.
 
-Either legacy email notification flags or Shoutrrr URLs can be used; however, directly using URLs is recommended for greater control and clarity, especially for configuring TLS settings (e.g., STARTTLS or Implicit TLS).
+!!! Warning "Deprecated"
+    Legacy email notification flags (e.g., `--notification-email-from`, `--notification-email-to`, `--notification-email-server`) are **deprecated**. Use `--notification-url` with an `smtp://` URL instead. See [Transitioning from Legacy Email Notifications to Shoutrrr](#transitioning-from-legacy-email-notifications-to-shoutrrr) below.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)